lib/internal/errors.js

```diff
@@ -786,6 +786,10 @@ E('ERR_FALSY_VALUE_REJECTION', function(reason) {
   this.reason = reason;
   return 'Promise was rejected with falsy value';
 }, Error);
+E('ERR_FEATURE_UNAVAILABLE_ON_PLATFORM',
+  (feature) => 'The feature ' + feature +
+    ' is unavailable to the current platform' +
```

Comment:
```suggestion
    ' is unavailable on the current platform' +
```
The next line - `, which is being used to run Node.js` - is arguably a little superfluous.

Reply:
You are right that is obvious that the current platform regards to the one which runs node.js; But I can Imagine a frustrated developer who just wants to understand why its recursive doesn't work, after he has ported his app which uses  fancy libraries which usually do cover fs.watch, to be benefited from this remark. I believe that an error should be crystal clear.